File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P08_EmployeeData/P08_EmployeeData.cs
using System;

namespace P08_EmployeeData
{
    class P08_EmployeeData
    {
        static void Main(string[] args)
        {
            string firstName = Console.ReadLine();
            string lastName = Console.ReadLine();
            byte age = byte.Parse(Console.ReadLine());
            char sex = char.Parse(Console.ReadLine());
            long id = long.Parse(Console.ReadLine());
            int employeeNumber = int.Parse(Console.ReadLine());

            Console.WriteLine($"First name: {firstName}");
            Console.WriteLine($"Last name: {lastName}");
            Console.WriteLine($"Age: {age}");
            if (sex == 'm' || sex == 'f')
            {
                Console.WriteLine($"Gender: {sex}");

            }
            Console.WriteLine($"Personal ID: {id}");
            Console.WriteLine($"Unique Employee number: {employeeNumber}");
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P08_EmployeeData.java
import java.util.Scanner;
public class P08_EmployeeData {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        String firstName = console.nextLine();
        String lastName = console.nextLine();
        byte age = Byte.parseByte(console.nextLine());
        char sex = console.nextLine().charAt(0);
        long id = Long.parseLong(console.nextLine());
        int employeeNumber = Integer.parseInt(console.nextLine());

        System.out.format("First name: %s%n", firstName);
        System.out.format("Last name: %s%n", lastName);
        System.out.format("Age: %d%n", age);
        if (sex == 'm' || sex == 'f')
        {
            System.out.format("Gender: %s%n", sex);

        }
        System.out.format("Personal ID: %d%n", id);
        System.out.format("Unique Employee number: %d%n", employeeNumber);
    }
}
